apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.Snippets/SpannerConnectionSnippets.cs

```diff
@@ -425,16 +425,44 @@ public async Task WhenSessionPoolReady()
             // End sample
         }
 
-        [Fact]
-        public void GetSessionPoolDatabaseStatistics()
+        [Fact]
+        public void GetSessionPoolStatistics()
         {
             string connectionString = _fixture.ConnectionString;
 
             // Sample: GetSessionPoolDatabaseStatistics
             // Additional: GetSessionPoolDatabaseStatistics
```

Comment:
You need to change these, else the docs won't pick it up properly.

```suggestion
            // Sample: GetSessionPoolStatistics
            // Additional: GetSessionPoolStatistics
```